CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+## Version 3.12.14
+
+_2022-09-20_
+
+* Fix: Don't leak HTTP headers when setting invalid value.
```

Comment:
Should this include a reference to the [Snyk information exposure page](https://security.snyk.io/vuln/SNYK-JAVA-COMSQUAREUPOKHTTP3-2958044)?

Reply:
Sure! I will add it.